<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUpWidget</class>
 <widget class="QWidget" name="SignUpWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-family: Arial;

color:black;
</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>790</width>
     <height>503</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>501</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;
border-style: outset;
    border-width: 0px;
    border-radius: 16px;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="headreLabel">
         <property name="styleSheet">
          <string notr="true">font: bold;
color:#0F7C8A;
font-family: Arial;
font-size: 30px;</string>
         </property>
         <property name="text">
          <string>Sign up</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="nameLineEdit">
         <property name="minimumSize">
          <size>
           <width>301</width>
           <height>41</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
color: black;
background-color: #E6E6E6;
border-style: outset;
border-width: 0px;
border-radius: 10px;
border-color:#E6E6E6;;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="mailLineEdit">
           <property name="minimumSize">
            <size>
             <width>301</width>
             <height>41</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit{
color: black;
background-color: #E6E6E6;
border-style: outset;
border-width: 0px;
border-radius: 10px;
border-color:#E6E6E6;;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="pswdLineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>301</width>
             <height>41</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit{
color: black;
background-color: #E6E6E6;
border-style: outset;
border-width: 0px;
border-radius: 10px;
border-color:#E6E6E6;;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="confirmPswdLineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>301</width>
             <height>41</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit{
color: black;
background-color: #E6E6E6;
border-style: outset;
border-width: 0px;
border-radius: 10px;
border-color:#E6E6E6;;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QRadioButton" name="memberRadioButton">
               <property name="text">
                <string>Member</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="adminRadiButton">
               <property name="text">
                <string>Admin</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>60</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Minimum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>60</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="createAccountButton">
             <property name="minimumSize">
              <size>
               <width>301</width>
               <height>41</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">
border-style: outset;
border-width: 0px;
border-radius: 10px;
border-color:#D68243;;
padding: 0 8 px;
background:#D68243;
font: bold 14px;
color:white;
</string>
             </property>
             <property name="text">
              <string>Creat an account</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>128</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Already a member?</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="logInButton">
             <property name="maximumSize">
              <size>
               <width>59</width>
               <height>29</height>
              </size>
             </property>
             <property name="text">
              <string>Log In</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="picLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>2</horstretch>
        <verstretch>5</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>381</width>
        <height>501</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>381</width>
        <height>501</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
border-style: outset;
    border-width: 2px;
border-color:white;
    border-radius: 16px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
